src/diffusers/loaders/single_file_model.py

```diff
@@ -67,6 +67,17 @@
 }
 
 
+def _get_single_file_loadable_mapping_class(cls):
+    diffusers_module = importlib.import_module(__name__.split(".")[0])
```

Comment:
Thanks. I think we just need to add `import importlib` into the file 👍🏽

Reply:
you are right